<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1214</width>
    <height>703</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#333333;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>-10</y>
      <width>611</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:36pt; font-weight:700; font-style:italic; text-decoration: underline; color:#CCCCCC;&quot;&gt;Ex 12: TABELA HASH ED2&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonBuscarNomeArquivo">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>70</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Franklin Gothic Demi</family>
      <pointsize>10</pointsize>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#CCCCCC;
border-color:rgb(149, 0, 255);
color:#333333;

border-radius:10px;</string>
    </property>
    <property name="text">
     <string>Buscar</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_EncontrarBS">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>200</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Franklin Gothic Demi</family>
      <pointsize>11</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#CCCCCC;
border-color:rgb(149, 0, 255);
color:#333333;

border-radius:10px;</string>
    </property>
    <property name="text">
     <string>Procura</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../Projetos do qt/icons/magnifier.png</normaloff>../../Projetos do qt/icons/magnifier.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>70</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Franklin Gothic Demi</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
color:#CCCCCC;
</string>
    </property>
    <property name="text">
     <string>Nome do arquivo:</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>691</width>
      <height>461</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Franklin Gothic Demi</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#CCCCCC;
border-color:rgb(149, 0, 255);
color:#333333;

</string>
    </property>
    <column>
     <property name="text">
      <string>Matricula</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Nome</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Dep</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Titulação</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tipo de Contrato</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pushButtonMostrarDados">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>110</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Franklin Gothic Demi</family>
      <pointsize>11</pointsize>
      <italic>true</italic>
      <underline>false</underline>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#CCCCCC;
border-color:rgb(149, 0, 255);
color:#333333;

border-radius:10px;</string>
    </property>
    <property name="text">
     <string>MOSTRAR DADOS</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../Projetos do qt/icons/show.png</normaloff>../../Projetos do qt/icons/show.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNomeDoArquivo">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>70</y>
      <width>401</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cairo black</family>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#CCCCCC;
border-color:rgb(149, 0, 255);
color:#333333;

border-radius:10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_RemoverBS">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>200</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Franklin Gothic Demi</family>
      <pointsize>11</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#CCCCCC;
border-color:rgb(149, 0, 255);
color:#333333;

border-radius:10px;</string>
    </property>
    <property name="text">
     <string>Remoção</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../Projetos do qt/icons/remove.png</normaloff>../../Projetos do qt/icons/remove.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_InserirBS">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>290</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Franklin Gothic Demi</family>
      <pointsize>11</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#CCCCCC;
border-color:rgb(149, 0, 255);
color:#333333;

border-radius:10px;</string>
    </property>
    <property name="text">
     <string>Inserção</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../Projetos do qt/icons/add.png</normaloff>../../Projetos do qt/icons/add.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_AlterarBS">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>290</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Franklin Gothic Demi</family>
      <pointsize>11</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#CCCCCC;
border-color:rgb(149, 0, 255);
color:#333333;

border-radius:10px;</string>
    </property>
    <property name="text">
     <string>Alteração</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../Projetos do qt/icons/pen.png</normaloff>../../Projetos do qt/icons/pen.png</iconset>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>590</y>
      <width>301</width>
      <height>91</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="page"/>
    <widget class="QWidget" name="page_inserir">
     <widget class="QPushButton" name="pushButton_Inserir">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>301</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Franklin Gothic Demi</family>
        <pointsize>12</pointsize>
        <italic>true</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#CCCCCC;
border-color:rgb(149, 0, 255);
color:#333333;

border-radius:10px;</string>
      </property>
      <property name="text">
       <string>Inserir</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_alterar">
     <widget class="QPushButton" name="pushButton_Alterar">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>301</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Franklin Gothic Demi</family>
        <pointsize>12</pointsize>
        <italic>true</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#CCCCCC;
border-color:rgb(149, 0, 255);
color:#333333;

border-radius:10px;</string>
      </property>
      <property name="text">
       <string>Alterar</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_remover">
     <widget class="QPushButton" name="pushButton_Remover">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>301</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Franklin Gothic Demi</family>
        <pointsize>12</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#CCCCCC;
border-color:rgb(149, 0, 255);
color:#333333;

border-radius:10px;</string>
      </property>
      <property name="text">
       <string>Remover</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_procurar">
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <widget class="QPushButton" name="pushButton_Encontrar">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>301</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Franklin Gothic Demi</family>
        <pointsize>12</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#CCCCCC;
border-color:rgb(149, 0, 255);
color:#333333;

border-radius:10px;</string>
      </property>
      <property name="text">
       <string>Procurar</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget_2">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>380</y>
      <width>301</width>
      <height>91</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page_0"/>
    <widget class="QWidget" name="page_Nome_Matricula">
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>261</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Franklin Gothic Demi</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#CCCCCC;
</string>
      </property>
      <property name="text">
       <string>Nome</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Nome">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>301</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Franklin Gothic Demi</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#CCCCCC;
border-color:rgb(149, 0, 255);
color:#333333;

border-radius:10px;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>301</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Franklin Gothic Demi</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#CCCCCC;
</string>
      </property>
      <property name="text">
       <string>Matricula</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Mat">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>301</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Franklin Gothic Demi</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#CCCCCC;
border-color:rgb(149, 0, 255);
color:#333333;

border-radius:10px;</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>171</width>
      <height>211</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../Pictures/assinaturas-escolas-pucgoias.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>20</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Franklin Gothic Demi</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#CCCCCC;</string>
    </property>
    <property name="text">
     <string>Brunno Pedrosa Mesquita</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget_3">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>469</y>
      <width>301</width>
      <height>131</height>
     </rect>
    </property>
    <widget class="QWidget" name="page_2"/>
    <widget class="QWidget" name="page_ativado">
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>301</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Franklin Gothic Demi</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#CCCCCC;
</string>
      </property>
      <property name="text">
       <string>Departamento</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_Dep">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>301</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Franklin Gothic Demi</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#CCCCCC;
border-color:rgb(149, 0, 255);
color:#333333;

border-radius:10px;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>301</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Franklin Gothic Demi</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#CCCCCC;
</string>
      </property>
      <property name="text">
       <string>Titulação</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_Tit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>301</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Franklin Gothic Demi</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#CCCCCC;
border-color:rgb(149, 0, 255);
color:#333333;

border-radius:10px;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>80</y>
        <width>301</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Franklin Gothic Demi</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#CCCCCC;
</string>
      </property>
      <property name="text">
       <string>Tipo de Contrato</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_Tip">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>100</y>
        <width>301</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Franklin Gothic Demi</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#CCCCCC;
border-color:rgb(149, 0, 255);
color:#333333;

border-radius:10px;</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1214</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#CCCCCC;</string>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="styleSheet">
    <string notr="true">color:#CCCCCC;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
